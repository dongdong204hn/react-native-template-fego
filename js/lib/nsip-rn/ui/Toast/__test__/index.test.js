import React from 'react'
import Renderer from 'react-test-renderer'
import { shallow } from 'enzyme'
import ToastContainer from '../ToastContainer'

describe('Toast Tests', () => {
	it('renders correctly', () => {
		const tree = Renderer.create((
			<ToastContainer 
				key='toast'
				content='这是一个 toast 提示!!!'
				duration={3} />
		)).toJSON();
		
		expect(tree).toMatchSnapshot();
	});
})
