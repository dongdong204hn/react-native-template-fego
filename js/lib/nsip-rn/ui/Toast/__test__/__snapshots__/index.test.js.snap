// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toast Tests renders correctly 1`] = `
<View
  collapsable={undefined}
  pointerEvents="box-none"
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(0,0,0,0)",
      "bottom": 0,
      "justifyContent": "center",
      "left": 0,
      "opacity": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
      "zIndex": 1000,
    }
  }
>
  <View
    style={
      Array [
        Object {},
        Object {
          "alignItems": "center",
          "backgroundColor": "rgba(3, 3, 3, 0.7)",
          "borderRadius": 5,
          "minWidth": 100,
          "padding": 8,
        },
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFF",
          "fontSize": 14,
          "marginHorizontal": 10,
        }
      }
    >
      这是一个 toast 提示!!!
    </Text>
  </View>
</View>
`;
